x-pack/plugin/security/qa/smoke-test-all-realms/src/javaRestTest/resources/jwt/clientsecret.txt

```diff
@@ -0,0 +1 @@
+client-shared-secret-string
```

Comment:
I don't think this file is needed. right?

Reply:
Resolved. Removed file.